rules/autoconsent/sibbo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "sibbo",
+  "prehideSelectors": [".sibbo-layout, .sibbo-panel"],
```

Comment:
just .sibbo-layout seems to be enough

Reply:
.sibbo-layout only works on tested websites.